src/browser.ts

```diff
@@ -14,4 +14,8 @@ export interface DappeteerBrowser<Browser = unknown, Page = unknown>
   close(): Promise<void>;
 
   wsEndpoint(): string;
+
+  getUserDataDirPath(): string;
+
+  storeUserData(destination: string): boolean;
```

Comment:
How are we using the boolean returned from the method?

Reply:
It returns `true` if success `false` if fails somewhere
For future proofing and for users